systemvm/debian/opt/cloud/bin/setup/cloud-early-config

```diff
@@ -42,7 +42,14 @@ hypervisor() {
   grep -q QEMU /var/log/messages && echo "kvm" && return 0
 
   [ -d /proc/xen ] && mount -t xenfs none /proc/xen
-  [ -d /proc/xen ] && echo "xen-domU" && return 0
+  if [ -d /proc/xen ]; then
+    `dmesg | grep -q "Xen HVM"`
```

Comment:
For consistency, can you use this instead $(command here)?

Reply:
sure.